fission/src/mirabuf/MirabufLoader.ts

```diff
@@ -1,26 +1,125 @@
 import { mirabuf } from "../proto/mirabuf"
 import Pako from "pako"
-// import * as fs from "fs"
+
+const robots = "Robots"
+const fields = "Fields"
```

Comment:
I would change these variable names to indicate what they are doing. Like `robotsDir` or something like that. If there not doing anything special and it would just be `robotText` then we could probably just leave them as string literals.